hangman.py

```diff
@@ -1,213 +1,331 @@
+      False otherwise
+    '''
+    # & means intersection of these 2 sets
+    if set(secret_word) == set(secret_word) & set(letters_guessed):
+        return True
+    else:
+        return False
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    word = []
+    for letter in secret_word:
+        if letter in letters_guessed:
+            word += letter
+        else:
+            word += UNDERSCORE
+    return ' '.join(word)
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    alphabet = string.ascii_lowercase
+    new_alphabet = []
+    for letter in alphabet:
+        if letter not in letters_guessed:
+            new_alphabet += letter
+    return ''.join(new_alphabet)
+
+
+def victory(guesses: int, secret_word: str):
+    """
+    guesses: count of guesses left
+    secret_word: the word the user is guessing
+    returns: string with congratulations and total score
+    """
+    total_score = guesses * len(set(secret_word))
+    print(TextStyle.change_background_font(f'Congratulations! You won, the secret word was - {secret_word}'))
+    print(TextStyle.change_background_font(f'Your total score is {total_score}'))
+
+
+def letter_check(letter: str, guesses: int, warnings: int) -> tuple:
+    """
+    letter: entered letter by user
+    guesses: count of guesses left
+    warnings: count of warnings left
+    returns: which consist of letter, updated count of guesses, updated count of warnings
+    """
+
+    if letter.isalpha() and len(letter) == 1 or letter == HINT_SIGN:
+        value_correctness = 'Valid'
+        return letter, guesses, warnings, value_correctness
+    else:
+        if warnings <= 0:
+            guesses -= 1
+            if guesses <= 0:
+                value_correctness = 'Valid'
+                return letter, guesses, warnings, value_correctness
+        else:
+            warnings -= 1
+        if len(letter) != 1:
+            print(TextStyle.yellow(f'You must enter only 1 letter. {warnings} warnings left. {guesses} left.'))
+        elif letter.isalpha() is False:
+            print(TextStyle.yellow(
+                f'You must enter a letter of the Latin alphabet. Not a number or symbol. {warnings} warnings left {guesses} left.'))
+        else:
+            print(TextStyle.yellow(
+                f'Invalid value. Please enter 1 Latin letter. {warnings} warnings left {guesses} left.'))
+        value_correctness = 'Invalid'

```

Comment:
I see you use only 1 possible string value for that, so it makes sense a bit change the meaning to boolean value, with changing its name to `is_input_incorrect` for example

Reply:
Done